### PR TITLE
Add required decode for python 3 support

### LINKED ISSUE #242: tickets should be decoded on python 3
As seen from the recent quicket hook posts

TypeError at /tickets/quicket_hook/
the JSON object must be str, not 'bytes'

### PR DESCRIPTION
**Pull request by drnlm**
Should close #242

### CODE REVIEW

File: wafer/tickets/views.py [hunk 1]
```diff
@@ -75,7 +75,8 @@ def quicket_hook(request):
     if request.GET.get('secret') != settings.WAFER_TICKETS_SECRET:
         raise PermissionDenied('Incorrect secret')
 
-    payload = json.load(request)
+    # This is required for python 3, and in thweory fine on python 2
```
Comment (hodgestar) [inline wafer/tickets/views.py]: s/thw/th/ :)

**Comment by hodgestar** [comment]
Other than minor typo in comment, looks good to me. :)

**Comment by hodgestar** [comment]
:+1: if this has been tested somehow.

**Comment by drnlm** [comment]
I've added a test case, and we've ow had one successful post from quicket on the za.pycon.org site, so I believe the fix is now correct.

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**